src/GitHubExtension/DeveloperId/LoginUIController.cs

```diff
@@ -25,378 +39,280 @@ public ProviderOperationResult Initialize(IExtensionAdaptiveCard extensionUI)
-                {
-                    ""width"": 1
-                }
-            ],
-            ""rows"": [
-                {
-                    ""type"": ""TableRow"",
-                    ""cells"": [
+
+                        if (enterprisePATPageActionPayload.IsUrlAction())
                         {
-                            ""type"": ""TableCell"",
-                            ""items"": [
+                            Log.Logger()?.ReportInfo($"Create PAT Link clicked");
+
+                            try
+                            {
+                                Uri uri = new Uri(enterprisePATPageActionPayload?.URL ?? string.Empty);
+
+                                var browserLaunch = false;
+
+                                _ = Task.Run(async () =>
                                 {
-                                    ""type"": ""ActionSet"",
-                                    ""actions"": [
-                                        {
-                                            ""type"": ""Action.Submit"",
-                                            ""title"": """ + $"{loader.GetString("LoginUI_LoginPage_Button1Text")}" + @""",
-                                            ""tooltip"": """ + $"{loader.GetString("LoginUI_LoginPage_Button1ToolTip")}" + @""",
-                                            ""style"": ""positive"",
-                                            ""isEnabled"": true,
-                                            ""id"": ""Personal""
-                                        }
-                                    ],
-                                    ""horizontalAlignment"": ""Center"",
-                                    ""spacing"": ""None""
-                                }
-                            ],
-                            ""isVisible"": true,
-                            ""verticalContentAlignment"": ""Center"",
-                            ""height"": ""stretch"",
-                            ""spacing"": ""None"",
-                            ""horizontalAlignment"": ""Center""
+                                    // Launch GitHub login page on Browser.
+                                    browserLaunch = await Windows.System.Launcher.LaunchUriAsync(uri);
+
+                                    if (browserLaunch)
+                                    {
+                                        Log.Logger()?.ReportInfo($"Uri Launched to {uri.AbsoluteUri} - Check browser");
+                                    }
+                                    else
+                                    {
+                                        Log.Logger()?.ReportError($"Uri Launch failed to {uri.AbsoluteUri}");
+                                    }
+                                });
+                            }
+                            catch (UriFormatException ufe)
+                            {
+                                Log.Logger()?.ReportError($"Error: {ufe}");

```

Comment:
Consider displaying Failure pages for these exceptions

Reply:
Probably not needed, since this is just exceptions for URL click, which isn't blocking.